frontend/apps/reader/modules/readermenu.lua

```diff
@@ -262,10 +262,16 @@ function ReaderMenu:exitOrRestart(callback)
     end
 end
 
-function ReaderMenu:onShowReaderMenu()
+function ReaderMenu:onShowReaderMenu(is_show_main_tab)
+
```

Comment:
Stray newline

Reply:
> I will also remove the param sent to the event handler as I think it is a little weird

It's not that weird. You could even name it `show_tab_index`, so you could send `1` for left tab, `-1` for right tab, and have it generic here (who said the right tab is the _main_ one :)